jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3827,7 +3827,51 @@ static String valueOfCodePoint(int codePoint) {
 		}
 		return string;
 	}
-	/*[ENDIF] Java11*/
+
+
+	/**
+	 * Returns a string whose value is the concatenation of this string repeated
+	 * count times. 
+	 * 
+	 * @param count
+	 *            a positive integer indicating the number of times to be repeated
+	 * @return a string whose value is the concatenation of this string repeated count times
+	 * @throws IllegalArgumentException
+	 *             if the count is negative
+	 * @since 11
+	 */
+	public String repeat(int count) {
+		if (count < 0) {
+			throw new IllegalArgumentException();
+		}
+		if (count == 1) {
+			return this;
+		}
+
+		int length = this.lengthInternal();
+		int repeatlen = length * count;
+
+		if (enableCompression && (null == compressionFlag || this.coder == LATIN1)) {
```

Comment:
For consistency with all the other code in String.java I suggest we avoid the `this.` prefix for fields unless there is ambiguity.